StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/CustomerSheet/CustomerSheet+PaymentMethodAvailability.swift

```diff
@@ -54,3 +57,34 @@ extension Array where Element == STPPaymentMethodType {
         return pmTypes
     }
 }
+
+extension Array where Element == String {
+    func customerSheetSupportedPaymentMethodTypes() -> Result<[STPPaymentMethodType]?, Error> {
```

Comment:
I don't think this makes sense as a method on [String], it feels too broad. Like, it shouldn't show up in the autocomplete list when I'm dealing with arrays of Strings. Can we find a different place to put this?
- Can we add a little docstring on what this does

Reply:
Added as a static func on `CustomerSheet` via an extension & Added doc string.  Thanks!